oak_runtime/src/node/mod.rs

```diff
@@ -32,10 +32,13 @@ mod wasm;
 
 /// Trait encapsulating execution of a Node or pseudo-Node.
 pub trait Node: Send {
-    /// Return a name for this type of Node.
+    /// Returns a name for this type of Node.
     fn node_type(&self) -> &'static str;
 
-    /// Execute the Node, using the provided `Runtime` reference and initial handle.  The method
+    /// Returns a value indicating whether the Node can communicate externally.
+    fn external_facing(&self) -> bool;
```

Comment:
Could reduce the boilerplate by having a default trait `impl` that returns `true` (which is also a fail-safe value), and then just have an explicit `false` implementation for `WasmNode` (and test nodes).

Reply:
Good idea, thanks. Done.